src/components/ContactDetails/ContactDetailsAvatar.vue

```diff
@@ -362,9 +362,10 @@ export default {
 					// refresh view
 					// FIXME: not working
 					const key = contactId + '~' + addressbookId
-					const updated = this.$store.getters.getContact(key)
-					this.$store.dispatch('fetchFullContact', { contact: updated })
-					this.$emit('updateLocalContact', updated)
+					const testcontact = this.$store.getters.getContact(key)
+					await this.$emit('updateLocalContact', testcontact)
+					if (testcontact === this.$store.getters.getContact(key)) { console.error('contact is unchanged :-/') }
```

Comment:
you cannot check like this.
The reference can be different but the content the same